component/go.mod

```diff
@@ -4,9 +4,8 @@ go 1.18
 
 require (
 	github.com/stretchr/testify v1.8.1
-	go.opentelemetry.io/collector v0.68.0
-	go.opentelemetry.io/collector/confmap v0.68.0
-	go.opentelemetry.io/collector/consumer v0.68.0
```

Comment:
This one was removed, I guess that means that the module was not tidied before. This should not block the PR, but it's something we should investigate.